### DISCUSSION TITLE
EQ7 MyOpenMath

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#329] [General]
Reviewer: @tdegeorge 

Questions 1-8 are quadratic inequalities, and 9-16 are rational.

**Comment by tdegeorge** [comment]
I like the groupings! Should we do this in all the homework?

**Comment by tdegeorge** [comment]
I think these look good....did you also want to include some questions that ask students to add/subtract rational expressions? I know it is not in the section (I think it is a pre-req), but was wondering if we want to include some in here too

**Comment by tdegeorge** [comment]
I also tried to search for some that asked them graph the solutions (like maybe on a number line or something) but couldn't find any good ones.

**Comment by tdegeorge** [comment]
Do we want any of these, for example, to start the homework?
<img width="514" alt="Screenshot 2024-07-03 at 1 54 03 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/143553140/f3915978-63f5-4b61-9f4e-4b792966aecc">

**Comment by tdegeorge** [comment]
I'm a little torn because the objective says they should also be able to express the solution graphically. I think we need to add some more questions around that idea too.
